### ISSUE TITLE
global stopTrigger does not work

### ISSUE THREAD

**Issue by sterres** [esmini/esmini#406]
Hi,
it seems like esmini ignores global stop trigger. I made 2 easy example scenarios. In the coverage file it is marked as supported.

One uses a simulation time 20s stop trigger of global level --> ignored and esmini continues simulation 
The other one has the same trigger but on act level --> works.
[stop trigger.zip](https://github.com/esmini/esmini/files/10920848/stop.trigger.zip)

BR
Simon

**Comment by eknabevcc** [comment]
I think the reason is the condition rule "equalTo". With this rule the condition will ONLY be true when simulationTime becomes exactly equal to specified time value.

This is perfectly OK if you have control over simulation time step. For example, try the following:

`./bin/esmini --window 60 60 800 400 --fixed_timestep 0.02 --osc global stop trigger 20s.xosc`

With timestep 0.02, for example, you will eventually hit exactly 20.0 seconds.

But with following launch command:
`./bin/esmini --window 60 60 800 400 --osc global stop trigger 20s.xosc`
 Here the timestep is non deterministic, esmini will step according to passed system time in order to keep up with real-time clock.

In conclusion the recommendation is to either:
- change condition rule into "greaterThan" (as in the act scenario file), or
- run with fixed timestep

**Comment by sterres** [comment]
thank you for the explanation - my bad. With fixed timestep it works as you explained. And the hint with greaterThan is also very helpful

**Comment by huber-florian** [comment]
Is there a possibility to start esmini from Python with --fixed_timestep 0.02 ?
Currently, I start it with "se.SE_Init(input_as_bytes, 0, 1, 0, 0)" and it may solve my problems.

Unfortunately, I have many problems with indeterministic behavior of esmini currently. Should the behavior be completely deterministic afterwards?
